extensions/ion/ion-client/src/main/java/org/eclipse/dataspaceconnector/ion/IonClientExtension.java

```diff
@@ -14,7 +15,7 @@
 
     @Override
     public Set<String> provides() {
-        return Set.of(IonClient.FEATURE, DidResolver.FEATURE);
+        return Set.of(IonClient.FEATURE, DidResolverRegistry.FEATURE);
```

Comment:
I suppose the `DidResolverRegistry` should be `required`, right? Since the `IonClientExtension` will likely register the `IonClient` with it. 
Then, this extension should `provide` both `IonClient.FEATURE` and `DidResolver.FEATURE`

Reply:
yes, but let's discuss the concept or provides and requires as a separate issue.

I didn't see any other comments but this could be because of the number of changes. Is there anything we need to address in this PR?